backend/canisters/group_index/impl/src/jobs/mod.rs

```diff
@@ -1,7 +1,7 @@
 use crate::RuntimeState;
 
-mod calculate_community_hotness;
 mod calculate_hot_groups;
```

Comment:
Can the `calculate_hot_groups` job be deleted now?

Reply:
Once the FE uses explore_groups we can of course delete "search" and "recommended_groups" and all supporting code including the calculate_hot_groups job.